### ISSUE TITLE
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult

### ISSUE THREAD

**Issue by reddy572** [PowerShell/PSReadLine#3014]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Exception

System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

### Screenshot

![Screenshot 2021-11-11 164253](https://user-images.githubusercontent.com/43253927/141373169-d90fa402-bf6f-4b7c-ba43-ab041a5e0870.png)

### Environment data

```console
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.19043
BufferWidth: 120
BufferHeight: 30
```

### Steps to reproduce

Update powershell 7.1.5 to 7.2.0

### Actual behavior

![Screenshot 2021-11-11 164253](https://user-images.githubusercontent.com/43253927/141373293-8481ec50-dd4e-4be2-ab0f-67c8168721b7.png)

**Comment by andriibudzan** [comment]
I have the same problem after updating PowerShell to 7.2.0 on Windows 10.0.22000

![image](https://user-images.githubusercontent.com/33166673/141374412-1ebad1ae-5df7-4498-b4da-a461eb0adb6f.png)

**Comment by stijnnel** [comment]
had the same error after upgrading to powershell 7.2.0. I uninstalled current version which was 2.2.0 beta1of PSreadline and installed the 2.2.0-beta4 preview. this version seems to work with no errors in 7.2.0.

**Comment by reddy572** [comment]
installed the 2.2.0-beta4 preview and it's working for me. Thanks for the help!

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
